examples/default.yaml

```diff
@@ -67,6 +67,10 @@ mounts:
   # 🔵 This file: true (only for "/tmp/lima")
   writable: true
 
+# Mount type for above  mounts, such as "sshfs" (from sshocker) or "9p" (from virtfs)
```

Comment:
```suggestion
# Mount type for above  mounts, such as "reverse-sshfs" (from sshocker) or "9p" (from QEMU’s virtio-9p-pci, aka virtfs)
```

Reply:
I should probably decouple the yaml/human names, from the mount types...